site/content/en/docs/tasks/integrate_a_custom_job.md

```diff
@@ -0,0 +1,47 @@
+---
+title: "Integrate a custom Job"
+date: 2022-07-25
+weight: 8
+description: >
+  Integrate a custom Job with Kueue.
+---
+# Integrate a custom Job with Kueue
+
+This page shows how to integrate a custom Job with Kueue. 
+
+## What we need to build
```

Comment:
nit: I'm not sure `build` is the right word here. If the integration is part of the Kueue repo, then its build system will take care. I would suggest rather `provide` or `specify`.

Reply:
Got it. I think you're right.